app/views/vendors/index.html.erb

```diff
@@ -18,8 +20,8 @@
         <img src="http://placehold.it/1000x1000&text=Thumbnail">
 
         <div class="panel">
-          <h5>Item Name</h5>
-          <h6 class="subheader">$000.00</h6>
+          <h5><%= link_to "#{vendor.vendor_name}", vendor_path(vendor) %></h5>
```

Comment:
You don't actually need to use string interpolation here - `vendor.vendor_name` will return a string, so you can just call that directly as the first argument here instead of interpolating it into a string!